src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -318,3 +324,13 @@ def test_set_get_vi_int_64_attribute(session):
     session.channels['0'].active_advanced_sequence_step = 1
     read_advanced_sequence_step = session.channels['0'].active_advanced_sequence_step
     assert read_advanced_sequence_step == 1
+
+
+def test_channel_format_types():
+    with nidcpower.Session('4162', [0, 1], False, 'Simulate=1, DriverSetup=Model:4162; BoardType:PXIe') as simulated_session:
```

Comment:
For the `channels` parameter on both `nidcpower` and `nifgen` constructor:

* Allow `None`
* Default channels to `None` and not `""`
* Add a test.

Reply:
Done